docs/ticket-triage.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Ticket Triage
+=============
+
+Ticket triage is a weekly, informal ritual that is meant to solve the problem of
+tickets getting opened and then forgotten about. It is simple and keeps project
+momentum going and prevents ticket cruft.
+
+It fosters conversation around project tasks and philosophies as they relate to 
+milestones.
+
+Process
+-------
+- The role of Ticket Triager rotates weekly 
+- The Triager needs admin status on ``Trac``
```

Comment:
Hm I made us all admins but most triage duties probably don't really require this.  Maybe creating new milestones requires "roadmap admin" or something like that ... but I'm also not sure creating new milestones is something that should be done by this person during the course of normal triage work.